client/sender-app/start/start.js

```diff
@@ -13,17 +13,22 @@
     vm.test = 'Start';
     vm.startGame = startGame;
     vm.userName = '';
+    vm.placeholder = 'What\'s yo name?';
     vm.btnShow = false;
+    vm.btnStart = false;
     vm.setUser = setUser;
 
     changeStateListener();
 
     // user setting their name and revealing the start game button
     function setUser(event) {
+      var userName = vm.userName;
       // set username in playerUser factory
       playerUser.setUser(vm.userName);
       // unhide btn
       vm.btnShow = true;
+      vm.userName = '';
```

Comment:
I'm not clear on why username is moving around so much in here--why push it to a local variable and why clear it from vm?